ocaml/sdk-gen/component-test/jsonrpc-client/go/client.go

```diff
@@ -0,0 +1,78 @@
+package main
```

Comment:
Better to put this into an `example` folder.

Reply:
This is a placeholder which would be updated by QA.